src/ui/src/components/blueprints/BlueprintsAutogen.vue

```diff
@@ -67,10 +74,14 @@ const tracking = useWriterTracking(wf);
 const isBusy = ref(false);
 const prompt = ref("");
 const errorMessage = ref<string | null>(null); // Track error messages
+const autogenController = ref<AbortController | null>(null);
```

Comment:
I think you don't need reactivity here

```suggestion
let autogenController: AbortController | null = null;
```